lib/vnet/diag/ssh.go

```diff
@@ -155,8 +181,8 @@ func (d *SSHDiag) openSSHConfigIncludesVNetSSHConfig(r io.Reader) (bool, error)
 
 // openSSHConfigLineIncludesPath returns true if the given line of an OpenSSH
 // configuration file is an include statement for the given path.
-func (d *SSHDiag) openSSHConfigLineIncludesPath(line, wantPath string) bool {
-	wantPath = d.normalizePath(wantPath)
+func (c *SSHConfigChecker) openSSHConfigLineIncludesPath(line, wantPath string) bool {
```

Comment:
This might be a good candidate for a fuzz test.

Reply:
added the fuzz test and oss-fuzz build